scripts/gym_bridge.py

```diff
@@ -107,8 +107,8 @@ def __init__(self):
         # ts.registerCallback(self.drive_callback)
 
         # Timer
-        self.timer = rospy.Timer(rospy.Duration(0.004), self.timer_callback)
-        self.drive_timer = rospy.Timer(rospy.Duration(0.05), self.drive_timer_callback)
+        self.timer = rospy.Timer(rospy.Duration(0.01), self.timer_callback)
```

Comment:
The first timer is used to update all the scan and odom information, this should be kept at 250 Hz.